### PR TITLE
[SD-3426] Order the results by name alphabetically:

### PR DESCRIPTION
**Pull request by sivakuna-aap**
[SD-3426](https://dev.sourcefabric.org/browse/SD-3426): Order the results by name alphabetically
Required by https://github.com/superdesk/superdesk/pull/1285

### CODE REVIEW

File: superdesk/publish/subscribers.py [hunk 1]
```diff
@@ -107,8 +109,6 @@ class SubscribersResource(Resource):
         }
     }
 
-    datasource = {'default_sort': [('_created', -1)]}
```
Comment (mugurrus) [inline superdesk/publish/subscribers.py]: This removes default sorting, the results will be unsorted

File: superdesk/roles/roles.py [hunk 2]
```diff
@@ -39,9 +41,7 @@ class RolesResource(Resource):
             'type': 'boolean'
         },
     }
-    datasource = {
```
Comment (mugurrus) [inline superdesk/roles/roles.py]: Same as above

File: superdesk/publish/subscribers.py (hunk 1 above)
Comment (mugurrus) [inline superdesk/publish/subscribers.py]: Sorry, I just noticed in the superdesk PR that you do the sorting from the client side.

**Comment by mugurrus** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**